packages/cli/src/services/project-updater.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from 'fs-extra'
+import * as path from 'path'
+
+export async function updatePackageJsonDependencyVersions(version: string, projectPath: string): Promise<void> {
+    const projectAbsolutePath = path.resolve(projectPath)
+    const packageJsonPath = path.join(projectAbsolutePath, 'package.json')
+    try {
+        const packageJsonContents = require(packageJsonPath)
+        packageJsonContents.dependencies['@boostercloud/framework-core'] = `^${version}`
+        if (packageJsonContents.dependencies['@boostercloud/framework-types'] !== undefined) {
+            packageJsonContents.dependencies['@boostercloud/framework-types'] = `^${version}`
```

Comment:
I'd extract these into `const`s instead of repeating them again and again

Reply:
Done! Nonetheless, I think we are about to avoid upgrading package.json in the end @NickSeagull